Makefile

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 DATE=$(shell date -u  +%Y-%m-%d)
-VERSION=$(shell cat VERSION | sed 's/[-dev]//g')
+VERSION=$(shell cat VERSION)
```

Comment:
Now also the ```-dev```suffix would be added, is this intended?

Reply:
I think that cicd calls the .ci/build with the appropriate version and `make build` is for local builds.

@ccwienk

Reply:
Let's stick with the suffix removal for now.

Reply:
Done.